engine/src/main/java/com/datatorrent/stram/StramClient.java

```diff
@@ -448,6 +450,14 @@ public void startApplication() throws YarnException, IOException
       }
       String libJarsCsv = copyFromLocal(fs, appPath, localJarFiles.toArray(new String[]{}));
 
+      if (UserGroupInformation.isSecurityEnabled() && customKeystoreConfig != null && customKeystorePath != null) {
+        String[] customSSLConfArr = {customKeystoreConfig, customKeystorePath};
+        String customSSLFileConf = copyFromLocal(fs, appPath, customSSLConfArr);
+        String configFileName = (new java.io.File(customKeystoreConfig)).getName();
+        dag.getAttributes().put(Context.DAGContext.STRAM_HTTP_CUSTOM_CONFIG, configFileName);
```

Comment:
@tweise HTTP_ by itself (without any other decoration), even under DAGContext, doesn't capture and convey to the user what the attribute is for? Management, webservice or STRAM are better monikers. 

I also disagree with both characterizations that it would be "wrong" to call it STRAM, whilst I do agree there are better names for it and second that consistency can be sacrificed without any thought when a better option is available as that could end up confusing the user. It is important to think about it on a case by case basis and weigh the pros and cons before coming to a final decision.

I would second your suggestion to call out API changes explicitly, have more than one committer review API changes and would recommend adding it to the guidelines, though I don't understand the use of the word "qualified". Aren't all committers qualified by definition? Are you suggesting to break up committers into groups? In reality, what I have experienced in general, is that where there are PRs with API changes, there are more than one committers looking and commenting on it because of the general concern around API. I think you would be hard pressed to find PRs with API changes, where the specific API changes weren't discussed on the dev list or reviewed by multiple committers in the PR.